systemtests/twohosts/sanity_test.go

```diff
@@ -1352,3 +1352,163 @@ func TestTwoHostsVxlanAddDelNetworkConsul_sanity(t *testing.T) {
 		t.Fatalf("Error - network %s doesn't seem to be deleted \n", "green")
 	}
 }
+
+func TestTwoHostsSingleVlanPingSuccessMultiAddDelEp_sanity(t *testing.T) {
+	defer func() {
+		utils.ConfigCleanupCommon(t, testbed.GetNodes())
+		utils.StopOnError(t.Failed())
+	}()
+
+	jsonCfg :=
+		`{
+        "Hosts" : [{
+            "Name"                      : "host1",
+            "Intf"                      : "eth2"
+        },
+        {
+            "Name"                      : "host2",
+            "Intf"                      : "eth2"
+        }],
+        "Tenants" : [ {
+            "Name"                      : "tenant-one",
+            "DefaultNetType"            : "vlan",
+            "SubnetPool"                : "11.1.0.0/16",
+            "AllocSubnetLen"            : 24,
+            "Vlans"                     : "11-48",
+            "Networks"  : [ {
+                "Name"                  : "orange",
+                "Endpoints" : [
+                {
+                    "Container"         : "myContainer1",
+                    "Host"              : "host1"
+                },
+                {
+                    "Container"         : "myContainer2",
+                    "Host"              : "host2"
+                } ]
+            } ]
+        } ]
+        }`
+
+	utils.ConfigSetupCommon(t, jsonCfg, testbed.GetNodes())
+
+	node1 := testbed.GetNodes()[0]
+	node2 := testbed.GetNodes()[1]
+	for i := 0; i < 10; i++ {
```

Comment:
just wondering if we should also test deletion of just one (random?) endpoint, instead of cleaning up all and recreating them?